# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/python-josepy/no-install-deps.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

Prevent it from downloading stuff from the internet.

--- josepy-1.13.0/setup.py.vanilla	2024-01-07 17:13:49.965440977 +0100
+++ josepy-1.13.0/setup.py	2024-01-07 17:14:00.583115929 +0100
@@ -6,15 +6,6 @@
 
 # Please update tox.ini when modifying dependency version requirements
 install_requires = [
-    # load_pem_private/public_key (>=0.6)
-    # rsa_recover_prime_factors (>=0.8)
-    # add sign() and verify() to asymetric keys (RSA >=1.4, ECDSA >=1.5)
-    'cryptography>=1.5',
-    # Connection.set_tlsext_host_name (>=0.13)
-    'PyOpenSSL>=0.13',
-    # For pkg_resources. >=1.0 so pip resolves it to a version cryptography
-    # will tolerate; see #2599:
-    'setuptools>=1.0',
 ]
 
 testing_requires = [
